src/plugins/saved_objects_management/public/lib/process_import_response.ts

```diff
@@ -30,6 +31,7 @@ export interface FailedImport {
   obj: Pick<SavedObjectsImportError, 'id' | 'type' | 'title'>;
   error:
     | SavedObjectsImportConflictError
+    | SavedObjectsImportAmbiguousConflictError
```

Comment:
Rather than try to exclude the `SavedObjectsImportAmbiguousConflictError` in this client-side code, I went ahead and added it here to satisfy the type checker. Additional client-side changes will be needed in a future PR to make the saved objects management page behave properly.